package.json

```diff
@@ -110,7 +110,7 @@
     "zeebe-bpmn-moddle": "^0.15.0"
   },
   "peerDependencies": {
-    "@bpmn-io/properties-panel": "^0.20.0",
+    "@bpmn-io/properties-panel": "^0.20.1",
```

Comment:
Do we need to be that strict, can't we simply specify `20.x`?

Reply:
`^0.20.1` is in fact 0.x but not lower than 0.20.1, cf. https://docs.npmjs.com/about-semantic-versioning

Reply:
That's true, but not consistent with how we handle the others. I missed it as well